code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/azure/AzureIdentityProvider.java

```diff
@@ -69,7 +69,7 @@
 
     @Override
     public Set<Identity> getIdentities(Account account) {
-        if (!isConfigured() || !AzureConstants.CONFIG.equalsIgnoreCase(SecurityConstants.AUTH_PROVIDER.get()) ||
+        if (!isConfigured() ||
```

Comment:
isConfigured() already checks for this. This was your review comment on the last PR.